cmd/system-probe/app/config.go

```diff
@@ -8,49 +8,19 @@ package app
 import (
 	"fmt"
 
+	cmdconfig "github.com/DataDog/datadog-agent/cmd/agent/common/commands/config"
```

Comment:
I think the reason I didn't do this to begin with was importing from another `cmd` felt strange from a code structure  point of view. If we are OK with this, then this PR looks good.

Reply:
Yes, I had the same felling but I saw this pattern in other agents so I guess that is OK
https://github.com/DataDog/datadog-agent/blob/a698aec1367038765512645ab87faf087f14244b/cmd/process-agent/main_common.go#L15